### PR TITLE
Feature/encrypted sessions

### PR DESCRIPTION
**Pull request by PhilippSpo**
add option to encrypt session files using the `encrypt` option.
This encrypts the session file symmetrically using the `sessionId` as the key and the node crypto module.

### CODE REVIEW

**Comment by valery-barysok** [comment]
great. Please, add option description to README.md file.

File: lib/session-file-helpers.js [hunk 1]
```diff
@@ -135,8 +139,17 @@ var helpers = {
     });
 
     operation.attempt(function () {
-      fs.readJson(sessionPath, 'utf8', function (err, json) {
-        if (!err) return callback(null, helpers.isExpired(json, options) ? null : json);
+      fs.readFile(sessionPath, 'utf8', function (err, data) {
+
+        if (!err) {
+          var json
```
Comment (valery-barysok) [inline lib/session-file-helpers.js]: replace with 

    var json = JSON.parse(options.encrypt ? this.decrypt(data, sessionId) : data);

Comment (PhilippSpo) [inline lib/session-file-helpers.js]: :+1:

**Comment by valery-barysok** [comment]
@PhilippSpo, thanks. I will merge and publish it today later.

**Comment by valery-barysok** [comment]
@PhilippSpo, I published new 0.0.23 version with your changes.

**Comment by TYoung86** [comment]
This is not encryption, this is "encraption". Since the key is the session id, and the file is named by the session id, you are not protecting the file by encrypting it. :(

**Pull request merged**